hnix-store-core/src/System/Nix/Internal/Hash.hs

```diff
@@ -0,0 +1,159 @@
+{-# LANGUAGE AllowAmbiguousTypes #-}
+{-# LANGUAGE DataKinds           #-}
+{-# LANGUAGE PolyKinds           #-}
+{-# LANGUAGE RankNTypes          #-}
+{-# LANGUAGE TypeFamilies        #-}
+{-# LANGUAGE KindSignatures      #-}
+{-# LANGUAGE ScopedTypeVariables #-}
+{-# LANGUAGE TypeApplications    #-}
+
+module System.Nix.Internal.Hash where
+
+import qualified Crypto.Hash.MD5       as MD5
+import qualified Crypto.Hash.SHA1      as SHA1
+import qualified Crypto.Hash.SHA256    as SHA256
+import qualified Data.ByteString       as BS
+import qualified Data.ByteString.Char8 as BSC
+import           Data.Bits             (xor)
+import qualified Data.ByteString       as BS
+import qualified Data.ByteString.Lazy  as BSL
+import qualified Data.Hashable         as DataHashable
+import           Data.List             (foldl')
+import           Data.Proxy            (Proxy(Proxy))
```

Comment:
Unnecessary import?

Reply:
At the time it was leftover. Now it's needed for a use of `natVal`.